Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.py

```diff
@@ -0,0 +1,862 @@
+"""TOPdesk integration for Cortex XSOAR"""
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import os
+import math
+import shutil
+import urllib3
+import traceback
+import dateparser
+
+from typing import Any, Dict, List, Optional, Callable, Tuple, Union
+from base64 import b64encode
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+INTEGRATION_NAME = 'TOPdesk'
+XSOAR_ENTRY_TYPE = 'Automation'  # XSOAR
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+MAX_API_PAGE_SIZE = 10000
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the TOPdesk service API"""
+
+    def __init__(self, base_url, verify, headers, new_query):
+        super().__init__(base_url=base_url, verify=verify, headers=headers)
+        self._proxies = handle_proxy(proxy_param_name='proxy', checkbox_default_value=False)
+        self.new_query = new_query
+
+    def get_list_with_query(self, list_type: str, start: Optional[int] = None, page_size: Optional[int] = None,
+                            query: Optional[str] = None, modification_date_start: Optional[str] = None,
+                            modification_date_end: Optional[str] = None) -> List[Dict[str, Any]]:
+        """Get list of objects that support start, page_size and query arguments."""
+
+        allowed_list_type = ["persons", "operators", "branches", "incidents"]
+        if list_type not in allowed_list_type:
+            raise ValueError(f"Cannot get list of type {list_type}.\n "
+                             f"Only {allowed_list_type} are allowed.")
+
+        url_suffix = f"/{list_type}"
+        inline_parameters = False
+        request_params: Dict[str, Any] = {}
+        if start:
+            url_suffix = f"{url_suffix}?start={start}"
+            inline_parameters = True
+        if page_size:
```

Comment:
```suggestion

        if page_size:
```
drop a line between if blocks - this will make your code much more readable and easier to actually look at.
This true for all if blocks.